plugin/ToolBar.qml

```diff
@@ -35,6 +35,8 @@ PanelBackground {
             autoHideTimer.start()
         } else {
             autoHideTimer.stop()
+            // Keep a transiting toolbar visible.
+            _active = (offset > 0)
```

Comment:
Sorry, I don't understand. I think it's the case with the first part of the ```if``` statement, isn't it ?

When ```autoShowHide``` becomes ```true``` and the toolbar is shown (*i.e.* ```_active``` is also ```true```), the timer is started.